src/main/java/com/esri/core/geometry/ogc/OGCLineString.java

```diff
@@ -120,7 +121,7 @@ public OGCPoint endPoint() {
 
 	@Override
 	public String geometryType() {
-		return "LineString";
+		return TYPE;
```

Comment:
This change is unrelated to the change that adds support for geometry collections. It would be helpful to move it into a separate commit.

Reply:
I call geometryType() in the implementation of isFlattened. I've changed the methods to avoid string compare.